idd/Energy+.idd.in

```diff
@@ -52242,7 +52242,7 @@ Coil:Cooling:DX:MultiSpeed,
        \memo fan operation. Requires two to four sets of performance data and will interpolate
        \memo between speeds. Modeled as a single coil (multi-speed compressor or multiple
        \memo compressors with row split or intertwined coil).
-       \min-fields 95
+       \min-fields 96
```

Comment:
96 min-fields for MultiSpeed coil? If this is a 2-spd coil do I really have to enter data for speed 3 and 4? A 2-spd coil ends at N34, plus A24, the previous alpha, gives 58 as min-fields for 2-spd. What am I missing?

Reply:
Make sense. I'll limit it to the first two speeds. That would be 59 (Numeric 34 and alpha 25).